apiary-metastore-listener/README.md

```diff
@@ -151,6 +151,14 @@ The following shows an example JSON message representing an "ALTER_PARTITION" ev
       "oldPartitionLocation": "s3://table_location/old_partition"
     }
 
+## Message Attributes
+The [Message attributes](https://docs.aws.amazon.com/sns/latest/dg/sns-message-attributes.html) enable filtering of the SNS events. This can be done by applying a filter policy in a subscription receivers.
+The following messages attributes are supported:
+
+|Field Name|Type|
+|----|----|
+|`eventType`|String|
```

Comment:
It would be nice to have a small description of the field just like the table above.

"One of: CREATE_TABLE, DROP_TABLE, ALTER_TABLE, ADD_PARTITION, DROP_PARTITION, ALTER_PARTITION"